src/main/java/io/specto/hoverfly/junit/core/model/GlobalActions.java

```diff
@@ -13,12 +13,16 @@
 package io.specto.hoverfly.junit.core.model;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import org.apache.commons.lang3.builder.EqualsBuilder;
 import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
 
 import java.util.List;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
If you want to keep the annotations go for it but I think it would be easier as a one liner on the object mapper. Either way, as long as it isn't configurable I think it's fine.